src/frontend/src/common/components/library/data_subview/components/RedirectConfirmationModal/index.tsx

```diff
@@ -19,11 +24,18 @@ const RedirectConfirmationModal = ({
   isOpen,
   onClose,
   onConfirm,
+  logoWidth,
+  logoHeight,
```

Comment:
prefer to set defaults here
```
logoHeight = 45,
```

Reply:
lol wow of course they should be there! Not really sure what i was thinking

Reply:
```suggestion
  logoHeight = 45,
```